archive/j/javascript/oddNumbers.js

```diff
@@ -0,0 +1,15 @@
+function printOdd(num) {
+  for (let i=1; i<num; i++) {
+    if(i % 2 !== 0) {
+      console.log(i);
+    }
+  }
+}
+
+function printEven(num) {
+  for (let i=1; i<num; i++) {
+    if(i % 2 === 0) {
+      console.log(i);
+    }
+  }
+}
```

Comment:
The requirements state that this should be an executable script which accepts a single input. The functions are a fair start but there should be some simple main execution logic as well.